pkg/identity/identity.go

```diff
@@ -76,3 +78,15 @@ func (id *signingIdentity) Credentials() []byte {
 func (id *signingIdentity) Sign(message []byte) ([]byte, error) {
 	return id.sign(message)
 }
+
+func GetCertificate(f string) (*x509.Certificate, []byte, error) {
```

Comment:
Since the parameter is a file name to be read instead of bytes, I wonder if better naming would be `ReadCertificate`. This fits the naming scheme used by Go standard packages to read files, and reduces the scope for confusion with any functions that get certificates from different parameters.

```suggestion
func ReadCertificate(f string) (*x509.Certificate, []byte, error) {
```